source/computer/fedora-setup.rst

```diff
@@ -21,6 +21,10 @@ Fedora 配置指南
    安装 Fedora 也可以参考 https://techz.io/how-to-install-fedora/
    给出的详细步骤与图解。
 
+.. note::
+
+   经常重装系统的用户（如每半年一次），建议编写 Bash 脚本来自动安装需要的软件。
```

Comment:
I don't think it's a good suggestion. Writing scripts takes time, needs higher bash skills, and  may not work for new Linux versions.

Reply:
Just put those command in a bash script,.difficult?

Reply:
It depends. Linux keeps changing. If one command fails, running the subsequent commands may be damaging.

Reply:
But we are reinstalling the system instead of updating it. Nothing is in our computer. Meanwhile new babies will first install linux in cml. Only when they reinstall it for many times, they will use a script.

Reply:
They will know how to save time if they really want to reinstall their system many times.

What I really don't like is, the sentence reads like a warning "Linux is unstable. You may need to reinstall it every half an year."

Reply:
Okay. Removed in ed62303.